clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -278,6 +278,9 @@ SYCL::getDeviceLibraries(const Compilation &C, const llvm::Triple &TargetTriple,
   if (TargetTriple.isNVPTX() && IgnoreSingleLibs)
     LibraryList.push_back(Args.MakeArgString("devicelib--cuda.bc"));
```

Comment:
We should remember to rename these libraries to something more akin to `devicelib-<triple>.bc` or (`sycl-devicelib-<triple>.bc`?) as we discussed

Reply:
That's a good point. We might as well do this in this patch already for AMD.
I can put up another one for the NVPTX one as well

Reply:
After a bit of consideration, I would leave it as is for now and put up a separate patch with those renamings for the NVPTX and AMD libaries later.

CMake renamings could cause dependency errors and in that case this entire patch would have to be reverted (As you pointed out)

Reply:
I agree :+1: